homeworks/egetoimre/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,51 @@
+user_info = {
+     "name": "Mike",
+     "age": 25,
+     "favourite_meals": [
+         "pizza",
+         "carbonara",
+         "sushi"
+     ],
+     "phone_contacts": {
+         "Mary": "+36701234567",
+         "Tim": "+36207654321",
+         "Tim2": "+36304567321",
+         "Jim": "+364005000"
+     }
+}
+
+#A 4 programozási nyelv bekérése, majd listába töltése
+programming_lang = input("Adjon meg 4 programozási nyelvet, vesszővel elválasztva, szóközök nélkül: ").split(",")
```

Comment:
Szia! Nem hiba, csak egy javaslat: ha több elemet vársz a felhasználótól, a string változó neve is lehetne többes számú, így leíróbb.